service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/devicetwin/DeviceTwin.java

```diff
@@ -493,6 +493,7 @@ private DeviceTwinDevice jsonToDeviceTwinDevice(String json) throws IOException
         deviceTwinDevice.setTags(twinState.getTags());
         deviceTwinDevice.setDesiredProperties(twinState.getDesiredProperty());
         deviceTwinDevice.setReportedProperties(twinState.getReportedProperty());
+        deviceTwinDevice.setCapabilities(twinState.getCapabilities());
```

Comment:
Is this feature released from service side?

Reply:
Yes

Reply:
Is this released for the code flow involving all the API's that call jsonToDeviceTwinDevice that exist here?